packages/dart/test/src/objects/parse_object/test_utils.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:parse_server_sdk/parse_server_sdk.dart';
```

Comment:
Isn't it better to put this file somewhere else?

Reply:
How about we put it in the root `test/ `so any util function can be written in it

What do you think?

Reply:
Yes, it is a good suggestion